docs/resources/Channel.md

```diff
@@ -1210,7 +1210,7 @@ Creates a new thread in a forum channel, and sends a message within the created
 - For the embed object, you can set every field except `type` (it will be `rich` regardless of if you try to set it), `provider`, `video`, and any `height`, `width`, or `proxy_url` values for images.
 - Examples for file uploads are available in [Uploading Files](#DOCS_REFERENCE/uploading-files).
 - Files must be attached using a `multipart/form-data` body as described in [Uploading Files](#DOCS_REFERENCE/uploading-files).
-- Note that when sending a message, you must provide a value for at **least one of** `content`, `embeds`, or `files[n]`.
+- Note that when sending a message, you must provide a value for at **least one of** `content`, `embeds`, `sticker_ids`, `components`, or `files[n]`.
```

Comment:
Is stickers being added here intentional?

Reply:
Yup, seems like it was just missing there before, it was already mentioned further down above the table.